src/main/java/org/jabref/gui/preferences/general/GeneralTab.java

```diff
@@ -49,6 +54,10 @@ public class GeneralTab extends AbstractPreferenceTabView<GeneralTabViewModel> i
     @FXML private Button autosaveLocalLibrariesHelp;
     @FXML private CheckBox createBackup;
     @FXML private TextField backupDirectory;
+    @FXML private CheckBox remoteServer;
+    @FXML private TextField remotePort;
+    @FXML private Button remoteHelp;
+    @Inject private FileUpdateMonitor fileUpdateMonitor;
```

Comment:
@Inject private BibEntryTypesManager entryTypesManager; is missing here

Reply:
Hello, can you give me more detailed advice? I'm not sure what you mean, do you want to delete this variable?